### PR TITLE
docs: Update LangChain integration to remove wrong links

### PR DESCRIPTION
**Pull request by jirispilka**
Update LangChain integration to remove wrong links and information about RAG-Web-Browser

@TC-MO please check English 🙏🏻

### CODE REVIEW

File: sources/platform/integrations/ai/langchain.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ In this example, we'll use the [Website Content Crawler](https://apify.com/apify
 Then we feed the documents into a vector index and answer questions from it.
 
 This example focuses on how to integrate Apify with LangChain using the Python language,
```
Comment (TC-MO) [inline sources/platform/integrations/ai/langchain.md:18]:
```suggestion
This example demonstrates how to integrate Apify with LangChain using the Python language.
```

File: sources/platform/integrations/ai/langchain.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ In this example, we'll use the [Website Content Crawler](https://apify.com/apify
...
-but if you prefer to use JavaScript, you can follow the same steps in the [JavaScript LangChain documentation](https://js.langchain.com/docs/modules/indexes/document_loaders/examples/web_loaders/apify_dataset).
+but if you prefer to use JavaScript, you can follow the same steps in the [JavaScript LangChain documentation](https://js.langchain.com/docs/integrations/document_loaders/web_loaders/apify_dataset/).
```
Comment (TC-MO) [inline sources/platform/integrations/ai/langchain.md:19]:
```suggestion
If you prefer to use JavaScript, you can follow the  [JavaScript LangChain documentation](https://js.langchain.com/docs/integrations/document_loaders/web_loaders/apify_dataset/).
```

File: sources/platform/integrations/ai/langchain.md [hunk 3]
```diff
@@ -122,11 +122,27 @@ answer: LangChain is a framework for developing applications powered by language
 source: https://python.langchain.com
 ```
 
-LangChain is a standard interface through which you can interact with a variety of large language models (LLMs). It provides modules you can use to build language model applications. It also provides chains and agents with memory capabilities.
+LangChain is a standard interface through which you can interact with a variety of large language models (LLMs).
+It provides modules you can use to build language model applications as well as chains and agents with memory capabilities.
+
+You can use all of Apify’s Actors as document loaders in LangChain.
+For example, to incorporate web browsing functionality, you can use the [RAG-Web-Browser](https://apify.com/apify/rag-web-browser) Actor.
```
Comment (TC-MO) [inline sources/platform/integrations/ai/langchain.md:129]:
```suggestion
For example, to incorporate web browsing functionality, you can use the [RAG-Web-Browser Actor](https://apify.com/apify/rag-web-browser).
```

File: sources/platform/integrations/ai/langchain.md [hunk 4]
```diff
@@ -122,11 +122,27 @@ answer: LangChain is a framework for developing applications powered by language
...
+```python
+loader = apify.call_actor(
+    actor_id="apify/rag-web-browser",
+    run_input={"query": "apify langchain web browser", "maxResults": 3},
+    dataset_mapping_function=lambda item: Document(page_content=item["text"] or "", metadata={"source": item["metadata"]["url"]}),
+)
+print("Documents:", loader.load())
+```
+
+In the similar way, you can use other Apify Actors to load data into LangChain and query the vector index.
```
Comment (TC-MO) [inline sources/platform/integrations/ai/langchain.md:142]:
```suggestion
Similarly, you can use other Apify Actors to load data into LangChain and query the vector index.
```

**Review by TC-MO — CHANGES_REQUESTED** [review]
Few changes , otherwise good to go! :)

**Comment by jirispilka** [comment]
@TC-MO Thank you!

**Review by TC-MO — APPROVED** [review]

**Pull request merged**